habit/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from datetime import datetime
+from django.contrib.auth.models import AbstractBaseUser
+# Create your models here.
+
+
+class User(AbstractBaseUser):
+    def __repr__(self):
+        return f"<User username={self.username}>"
+    
+    def __str__(self):
+        return self.username
+
+
+class Habit(models.Model):
+    name = models.CharField(max_length=50)
+    goal = models.CharField(max_length=200)
+    user = models.ForeignKey(User,on_delete=models.CASCADE,related_name="habit_user"
```

Comment:
for related name you want "habits" -- this is what lets you query for the habits that belong to a user, like `user.habits.all()`. If you keep this related name you'd have to type `user.habit_user.all()`